packages/app-mobile/components/NoteBodyViewer/NoteBodyViewer.tsx

```diff
@@ -24,24 +30,24 @@ interface Props {
 	onCheckboxChange?: HandleMessageCallback;
 	onRequestEditResource?: HandleMessageCallback;
 	onMarkForDownload?: OnMarkForDownloadCallback;
-	onScroll: HandleScrollCallback;
+	onScroll: (scrollTop: number)=> void;
 	onLoadEnd?: ()=> void;
+	pluginStates: PluginStates;
 }
 
+const logger = Logger.create('NoteBodyViewer');
+
 export default function NoteBodyViewer(props: Props) {
 	const dialogBoxRef = useRef(null);
 	const webviewRef = useRef<WebViewControl>(null);
 
-	const { html, injectedJs } = useSource(
-		props.noteBody,
-		props.noteMarkupLanguage,
-		props.themeId,
-		props.highlightedKeywords,
-		props.noteResources,
-		props.paddingBottom,
-		props.noteHash,
-		props.initialScroll,
-	);
+	useEffect(() => {
+		logger.debug('WebView load started');
+	}, []);
```

Comment:
Remove these debug log statements. While useful for debugging performance issues, they should no longer be necessary.